package.json

```diff
@@ -1,197 +1,199 @@
+                "type": "boolean",
+                "description": "Defines if bashdb output will be visible together with script output.",
+                "default": true
+              },
+              "trace": {
+                "type": "boolean",
+                "description": "Enable logging of the Debug Adapter Protocol.",
+                "default": true
+              }
+            }
+          }
+        },
+        "// Template for initial launch.json": null,
+        "// Added when: launch.json is missing && (": null,
+        "//   Add configuration... (in debug pane) -> Bash Debug ||": null,
+        "//   >Debug: Open launch.json -> Bash Debug ||": null,
+        "//   (>Debug: Start Debugging -> Bash Debug && file type is not shellscript)": null,
+        "// )": null,
+        "initialConfigurations": [
+          {
+            "type": "bashdb",
+            "request": "launch",
+            "name": "Bash-Debug (simplest configuration)",
+            "program": "${file}"
+          }
+        ],
+        "// Added when select 'Add configuration...' -> 'Bash Debug' in debug pane": null,
+        "// `label` is shown in IntelliSense and `name` is shown in DEBUG side-bar.": null,
+        "configurationSnippets": [
+          {
+            "label": "Bash-Debug (hardcoded script name)",
+            "description": "Debug a bash script (specify the path to the script in launch.json)",
+            "body": {
+              "type": "bashdb",
+              "request": "launch",
+              "name": "Bash-Debug (hardcoded script name)",
+              "cwd": "^\"\\${workspaceFolder}\"",
+              "program": "^\"\\${workspaceFolder}/path/to/script.sh\"",
+              "args": []
+            }
+          },
+          {
+            "label": "Bash-Debug (select script from list of sh files)",
+            "description": "Debug a bash script (select script from the list of sh files before starting debug)",
+            "body": {
+              "type": "bashdb",
+              "request": "launch",
+              "name": "Bash-Debug (select script from list of sh files)",
+              "cwd": "^\"\\${workspaceFolder}\"",
+              "program": "^\"\\${command:SelectScriptName}\"",
+              "args": []
+            }
+          },
+          {
+            "label": "Bash-Debug (type in script name)",
+            "description": "Debug a bash script (specify the script path before starting debug)",
+            "body": {
+              "type": "bashdb",
+              "request": "launch",
+              "name": "Bash-Debug (type in script name)",
+              "cwd": "^\"\\${workspaceFolder}\"",
+              "program": "^\"\\${command:AskForScriptName}\"",
+              "args": []
+            }
+          }
+        ],
+        "variables": {
+          "AskForScriptName": "extension.bash-debug.getProgramName",
+          "SelectScriptName": "extension.bash-debug.selectProgramName"
+        }
+      }
+    ]
+  }
 }
```

Comment:
Finding out changes (`"release": "standard-version"` was added) of `package.json` was a bit of a chore, please split commits into *addition* and *generate* (or *re-indent* and *addition*) from the next time, if possible.

Reply:
Whoops :). I have not noticed that the file was completey rewritten while installing dependnecy. 

I will be more careful in future.